src/eu/packages/eu-component-button/button.html.twig

```diff
@@ -0,0 +1,66 @@
+{#
+  Parameters:
+    - "variant" (string) (default: 'primary'): can be 'primary', 'secondary', 'call', 'ghost', 'search'
+    - "type" (string) (default: 'button'): can be the same type as HTML button - 'submit', 'reset', 'button'
+    - "icon" (associative array) (default: {}): predefined structure for Icon component
+    - "disabled" (bool) (default: false): Defined if button should be active or not (HTML disabled attribute)
+    - "icon_position" (string) (default: after): Define icon position, can be 'before' or 'after'
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (string) (default: '')
+        },
+        ...
+      ]
+  Blocks:
+    - "label"
+#}
+
+{# Internal properties #}
+
+{% set _css_class = 'ecl-button' %}
+{% set _extra_attributes = '' %}
+{% set _label = label|default('') %}
+{% set _variant = variant|default('primary') %}
+{% set _type = type|default('button') %}
```

Comment:
default button type should be set to submit

Reply:
Updated on https://github.com/ec-europa/ecl-twig/pull/14/commits/4693f1181179b688854415165e35c950d848ba3e

Reply:
And update tests in https://github.com/ec-europa/ecl-twig/pull/14/commits/45af0a88a12907129f82b9b8b86473f9c52f3815